ntbs-service/Pages/Notifications/Denotify.cshtml.cs

```diff
@@ -0,0 +1,74 @@
+using Microsoft.AspNetCore.Mvc;
+using ntbs_service.Models;
+using ntbs_service.Models.Enums;
+using ntbs_service.Pages_Notifications;
+using ntbs_service.Services;
+using System;
+using System.Threading.Tasks;
+
+namespace ntbs_service.Pages.Notifications
+{
+    public class DenotifyModel : NotificationModelBase
+    {
+        public ValidationService  ValidationService { get; set; }
+
+        [BindProperty]
+        public DenotificationDetails DenotificationDetails { get; set; }
+
+        [BindProperty]
+        public FormattedDate FormattedDateOfDenotification { get; set; }
```

Comment:
This is minor, but elsewhere we use "Formatted<Type>Date", so maybe we could rename to FormattedDenotificationDate for consistency?